secrets/store.go

```diff
@@ -80,8 +81,9 @@ func (s *Store) getSecrets() *Secrets {
 // but can still access the secrets as they were before Close is called.
 //
 // It's OK to call Close multiple times. Calls after the first one are no-ops.
-func (s *Store) Close() {
+func (s *Store) Close() error {
```

Comment:
Please add a line into the doc comment stating that this closer always returns nil error.